src/DataManagement/Saving/AlternativeSavingCalculator.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Linq;
+using System.Threading.Tasks;
```

Comment:
It could be unused libraries, but what uses `Threading.Tasks`?
`Diagnostics` seems to be unused too.

Reply:
You're correct, they were just unused dependencies, I removed them